docs/lesson-plans/week-7.md

```diff
@@ -0,0 +1,692 @@
+INSERT INTO other_table_name (columns, you, want, to, supply) VALUES
+    (columns, you, want, to supply),
+-- The first one is what entering null looks like. The last one is what a timestamp looks like.
+    (null, "hello", 5, "2020-08-26 10:20:00.918963-07");
+```
+
+An example using the `people` table:
+```sql
+-- Insert into our table from earlier.
+INSERT INTO people VALUES
+-- Usually we'd just want to let the DB manager enter the next SERIAL value for us.
+    (0, "Jumpstart Team", "Hey, this is the Jumpstart team, and we're really excited you could join us!", 0, 
+    CURRENT_TIMESTAMP, 'nice'),
+-- We think you're nice, but we'll leave that up to you :)
+    (1, "You", "Hey, put a description here!", 19, CURRENT_TIMESTAMP, null);
+
+-- Insert values into only the name and age columns. The id and signup_date will be handled for us, and bio will just be
+-- null.
+INSERT INTO people (name, age) VALUES
+    ("Your friend", 20);
+```
+
+>Remember, this is what the PERSONALITY column looks like
+>CREATE TYPE personality AS ENUM ('nice', 'mean', 'neither');
+
+#### The SELECT Statement
+
+The `SELECT` statement is the statement used for reading data that already exists (C**R**UD). This is probably the most 
+commonly used SQL statement, and we hope that you'll be pretty used to it by the end. There are many conditions and 
+extra things we can add on, like grouping, aggregate functions, renaming columns, sub-queries, joins, row limiting, 
+and many others, but to keep it simple right now we'll just work with the structure below. You'll learn more about some
+of those later on.
+
+Example of select statements:
+```sql
+-- Select all columns for all rows on a table. The bare minimum you need when selecting (the columns -- "*" = all -- and
+-- the table name).
+SELECT * FROM table;
+
+-- Select specific columns from a table with some condition(s).
+SELECT column, names, you, want, some_date_column_too
+    FROM table_name
+-- Note how equality comparison is done with a single "=". Even for string types.
+    WHERE names = 'someName'
+-- Can do comparisons with numbers.
+        AND column > 5
+-- Can do comparisons with dates too! Also, you can group conditions together if you want to using parentheses.
+        OR (some_date_column_too BEFORE CURRENT_TIMESTAMP
+-- Assuming the 'you' column is a VARCHAR/CHAR/TEXT type, you can search for strings that start with 'hello' using the 
+-- 'like' operator. This will find results like 'hello', 'hello ', and 'hello world'. The '%' is like a wildcard, it can
+-- match any string when used with like.
+            AND you LIKE "hello%")
+-- Check for null/not null with 'IS NULL' and 'IS NOT NULL'.
+        AND want IS NULL;
+    
+```
+
+An example using the `people` table:
+```sql
+-- Select all fields from every record in people.
+SELECT * FROM people;
+
+-- Select all fields from every record in people who are greater than 5.
+SELECT * FROM people 
+    WHERE age > 5;
```

Comment:
Explain that age is a column in the database so even though it was not passed as an argument SQL still knows where to look